packages/select/test/items.test.js

```diff
@@ -0,0 +1,76 @@
+import { expect } from '@esm-bundle/chai';
+import { fixtureSync } from '@vaadin/testing-helpers';
+import './not-animated-styles.js';
+import '../vaadin-select.js';
+
+describe('items', () => {
+  let select, overlay, listBox;
+
+  beforeEach(() => {
+    select = fixtureSync(`<vaadin-select></vaadin-select>`);
+    select.items = [{ label: 'Option 1', value: 'value-1' }];
```

Comment:
Not sure if it makes sense to test also that items are rendered in the case they have been set before initialization.